src/backend/data/post.js

```diff
@@ -39,6 +39,28 @@ function ensureFeed(feed) {
   return feed instanceof Feed ? Promise.resolve(feed) : Feed.byId(feed);
 }
 
+/**
+ * Determine the type of the post depending on the given URL.
```

Comment:
nit: I would say this is pretty obvious and the comment won't provide much value to the future developers.

Reply:
I would like to leave the comment that specifies the possible post types. I will just delete the line that you highlighted.